app/jenkins-for-jira-ui/src/components/ConnectionPanel/SetUpGuide.tsx

```diff
@@ -1,50 +1,33 @@
-import React, { useState } from 'react';
+import React from 'react';
 import { cx } from '@emotion/css';
-import Drawer from '@atlaskit/drawer';
 import PeopleGroup from '@atlaskit/icon/glyph/people-group';
 import Button from '@atlaskit/button/standard-button';
 import {
-	setUpGuideLink,
 	setUpGuideInfoPanel,
 	setUpGuideNestedOrderedList,
 	setUpGuideNestedOrderedListItem,
 	setUpGuideOrderedList,
 	setUpGuideOrderedListItem,
 	setUpGuideOrderListItemHeader,
 	setUpGuideParagraph,
-	setUpGuideUpdateAvailableHeader,
 	setUpGuideUpdateAvailableButtonContainer,
 	setUpGuideUpdateAvailableContent,
+	setUpGuideUpdateAvailableHeader,
 	setUpGuideUpdateAvailableIconContainer
 } from './ConnectionPanel.styles';
 import { JenkinsPluginConfig, JenkinsServer } from '../../../../src/common/types';
 import { UpdateAvailableIcon } from '../icons/UpdateAvailableIcon';
-
-type SetUpGuideLinkProps = {
-	onClick: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void,
-	label: string,
-};
-
-export const SetUpGuideLink = ({ onClick, label }: SetUpGuideLinkProps): JSX.Element => {
```

Comment:
Now that we know how we're going to use IPH I killed this off in favour of 'InProductHelpAction' that can be utilised anywhere in the app.